scout/server/blueprints/variants/templates/variants/str-variants.html

```diff
@@ -103,12 +105,13 @@
             <td>{{ variant.chromosome }}</td>
             <td>
               {{ variant.position }}
+              {{ reviewer_button(case,variant,case_groups,institute._id) }}
```

Comment:
This column looks like this:

<img width="324" alt="image" src="https://user-images.githubusercontent.com/28093618/187841388-4f213ac8-d099-4b31-8dd7-9f65da25682e.png">

Perhaps the 2 buttons can be placed on the same line, there is space!

Reply:
Right, I remember now - this looked wonky on one line when there was only the replacement "not configured" and "alignment missing" texts. I'll try with making those inactive buttons, with help popovers instead!

Reply:
They're a bit weird, but better than before!